news/atom.xml

```diff
@@ -14,8 +14,11 @@ layout:
     <name>The ev3dev team</name>
   </author>
 
+  {% assign index = 0 %}
```

Comment:
Why not use `forloop.index` instead of declaring your own index variable?

https://github.com/shopify/liquid/wiki/Liquid-for-Designers#helper-variables